### PR TITLE
Focus 6.0 patch

### PR DESCRIPTION
**Pull request by sblatz**
Adds strings for new 6.0 features

### CODE REVIEW

File: templates/focus-ios.xliff [hunk 1]
```diff
@@ -497,6 +594,10 @@
         <source>Your browsing history has been erased.</source>
         <note>Message shown after pressing the Erase button</note>
       </trans-unit>
+      <trans-unit id="URL.findOnPageLabel">
+        <source>Find in page: %@</source>
+        <note>Label displayed for find in page button when typing in the URL Bar</note>
```
Comment (Delphine) [inline templates/focus-ios.xliff]: Can you specificy what %@ will stand for here?

File: templates/focus-ios.xliff [hunk 2]
```diff
@@ -557,6 +658,18 @@
         <source>Example: searchengineexample.com/search/?q=%s</source>
         <note>Text displayed as an example of the template to add a search engine.</note>
       </trans-unit>
+      <trans-unit id="share">
+        <source>Share</source>
+        <note>Text for a share button</note>
+      </trans-unit>
+      <trans-unit id="share.trackerStatsLabel">
+        <source>%@ trackers blocked so far</source>
+        <note>Text used when the user shares their trackers blocked stats</note>
```
Comment (Delphine) [inline templates/focus-ios.xliff]: It's always good to say what a placeholder stands for so people know where to place it in the string once they translate (even though here it's easy to guess for me, some people have no clue). So like "%@ will be replaced by blablabla"

File: templates/focus-ios.xliff [hunk 3]
```diff
@@ -557,6 +658,18 @@
...
+      </trans-unit>
+      <trans-unit id="share.trackerStatsText">
+        <source>%1$@, the privacy browser from Mozilla, has already blocked %2$@ trackers for me. Fewer ads and trackers following me around means faster browsing! Get Focus for yourself here</source>
+        <note>The text shared to users after the user chooses to share there tracker stats</note>
```
Comment (Delphine) [inline templates/focus-ios.xliff]: Same comment about both placeholders.
Also not: in the note it says "there" --> should be "their"

File: templates/focus-ios.xliff [hunk 4]
```diff
@@ -25,9 +50,9 @@
       </trans-unit>
     </body>
   </file>
-  <file original="Blockzilla/Intro.strings" source-language="en" datatype="plaintext">
+  <file original="Blockzilla/en.lproj/Intro.strings" source-language="en" datatype="plaintext">
```
Comment (Delphine) [inline templates/focus-ios.xliff]: Unless I'm confused, the fact that the file name here changes will invalidate all the existing translations contained in that section (@flodolo to confirm).
I believe that's the only one file name change left to correct this time though.

Comment (flodolo) [reply templates/focus-ios.xliff]: Confirmed: if the file changes, the strings are removed. You can double check any of the complete locales, their translations are lost.

File: templates/focus-ios.xliff [hunk 5]
```diff
@@ -217,6 +258,10 @@
         <source>Cancel</source>
         <note>Label on button to cancel edits</note>
       </trans-unit>
+      <trans-unit id="Disabled">
+        <source>Disabled</source>
+        <note>label describing something as disabled</note>
```
Comment (flodolo) [inline templates/focus-ios.xliff]: Where are you using this? Most languages need to adapt adjectives to the name's gender, and this feels like a catch-all label, which is wrong.

File: templates/focus-ios.xliff [hunk 6]
```diff
@@ -225,6 +270,10 @@
         <source>Edit</source>
         <note>Label on button to allow edits</note>
       </trans-unit>
+      <trans-unit id="Enabled">
+        <source>Enabled</source>
+        <note>label describing something as enabled</note>
```
Comment (flodolo) [inline templates/focus-ios.xliff]: Same note as disabled.

File: templates/focus-ios.xliff [hunk 7]
```diff
@@ -293,6 +342,18 @@
         <source>Browse. Erase. Repeat.</source>
         <note>Second label for product description on the home screen</note>
       </trans-unit>
+      <trans-unit id="Intro.stackView">
+        <source>Intro.stackView</source>
+        <note>Accessibility identifier for onboarding page indicator stack view</note>
```
[the remark this answers is not in the document]
Comment (flodolo) [reply templates/focus-ios.xliff]: Agreed. This should either not be exposed at all to translation, or have a meaningful English sentence, with some explanation, given how technical it is.

File: templates/focus-ios.xliff [hunk 8]
```diff
@@ -393,6 +458,10 @@
         <source>Learn more.</source>
         <note>Subtitle for Send Anonymous Usage Data toggle on main screen</note>
       </trans-unit>
+      <trans-unit id="Settings.rate">
+        <source>Rate Firefox Focus</source>
```
Comment (flodolo) [inline templates/focus-ios.xliff]: You don't want to hard-code Firefox Focus, this should be a variable set a run-time (and set to Klar in some cases).

File: templates/focus-ios.xliff [hunk 9]
```diff
@@ -457,6 +530,14 @@
         <source>Embedded on sites to track your visits and to display functionality like share buttons</source>
         <note>Description for 'Block social Trackers'</note>
       </trans-unit>
+      <trans-unit id="Settings.toggleFaceID">
+        <source>Use Face ID to unlock app</source>
+        <note>Label for toggle on settings screen</note>
+      </trans-unit>
+      <trans-unit id="Settings.toggleFaceIDDescription">
+        <source>Face ID can unlock Firefox Focus if a URL is already open in the app</source>
```
Comment (flodolo) [inline templates/focus-ios.xliff]: Same issue: don't hardcode "Firefox Focus"

File: templates/focus-ios.xliff [hunk 10]
```diff
@@ -469,10 +550,22 @@
         <source>Send usage data</source>
         <note>Label for Send Usage Data toggle on main screen</note>
       </trans-unit>
+      <trans-unit id="Settings.toggleTouchID">
+        <source>Use Touch ID to unlock app</source>
+        <note>Label for toggle on settings screen</note>
+      </trans-unit>
+      <trans-unit id="Settings.toggleTouchIDDescription">
+        <source>Touch ID can unlock Firefox Focus if a URL is already open in the app</source>
```
Comment (flodolo) [inline templates/focus-ios.xliff]: Hard-coded "Firefox Focus".

File: templates/focus-ios.xliff (hunk 2 above)
Comment (flodolo) [reply templates/focus-ios.xliff]: This won't work. "X trackers" requires a proper plural form, which we currently don't support in XLIFF.
https://bugzilla.mozilla.org/show_bug.cgi?id=1181777

This should be something like `Trackers blocked: %@`

File: templates/focus-ios.xliff (hunk 3 above)
Comment (flodolo) [reply templates/focus-ios.xliff]: As above, this won't work with languages using complex plural forms.

Comment (sblatz) [reply templates/focus-ios.xliff]: Would love @brjones opinion on how to word this text that can be localized in a clean manner.

**Pull request closed without merging**